spec/system/courses/review_spec.rb

```diff
@@ -724,10 +724,160 @@
     expect(page).not_to have_content(course.name)
   end
 
-  scenario "school admin tries to access the review dashboard" do
-    sign_in_user school_admin.user, referrer: review_course_path(course)
+  context "when user is an admin" do
+    # Create a couple of passed submissions for the team 3.
+    let!(:submission_l1_t3) do
+      create(
+        :timeline_event,
+        :with_owners,
+        latest: true,
+        owners: [student_l3],
+        target: target_l1,
+        evaluator_id: team_coach.id,
+        evaluated_at: 4.days.ago,
+        passed_at: 1.day.ago
+      )
+    end
+    let!(:submission_l2_t3) do
+      create(
+        :timeline_event,
+        :with_owners,
+        latest: true,
+        owners: [student_l3],
+        target: target_l2,
+        evaluator_id: course_coach.id,
+        evaluated_at: 2.days.ago,
+        passed_at: nil,
+        created_at: 1.day.ago
+      )
+    end
+    let!(:team_submission) do
+      create(
+        :timeline_event,
+        :with_owners,
+        latest: true,
+        owners: team_l3.students,
+        target: team_target,
+        evaluator_id: course_coach.id,
+        evaluated_at: 1.day.ago,
+        passed_at: nil,
+        created_at: 2.days.ago
+      )
+    end
 
-    expect(page).to have_text("The page you were looking for doesn't exist!")
-    expect(page).not_to have_content(course.name)
+    # And one passed submission for team 2.
+    let!(:submission_l1_t2) do
+      create(
+        :timeline_event,
+        :with_owners,
+        latest: true,
+        owners: [student_l2],
+        target: target_l1,
+        evaluator_id: team_coach.id,
+        evaluated_at: 3.days.ago,
+        passed_at: 3.days.ago,
+        created_at: 4.days.ago
+      )
+    end
+
+    # Create a couple of pending submissions for the teams.
+    let!(:submission_l1_t1) do
+      create(
+        :timeline_event,
+        :with_owners,
+        latest: true,
+        target: target_l1,
+        reviewer: course_coach,
+        reviewer_assigned_at: 1.day.ago,
+        owners: [student_l1]
+      )
+    end
+    let!(:submission_l2_t2) do
+      create(
+        :timeline_event,
+        :with_owners,
+        latest: true,
+        target: target_l2,
+        owners: [student_l2],
+        created_at: 1.day.ago
+      )
+    end
+    let!(:submission_l3_t3) do
+      create(
+        :timeline_event,
+        :with_owners,
+        latest: true,
+        target: target_l3,
+        owners: [student_l3],
+        created_at: 2.days.ago
+      )
+    end
+
+    let!(:feedback) do
+      create(
+        :startup_feedback,
+        faculty_id: course_coach.id,
+        timeline_event: submission_l2_t3
+      )
+    end
+
+    scenario "can view all the submissions", js: true do
```

Comment:
You can't start a scenario description with _can_! The full test scenario would read weird... _"when user is an admin can view all the submissions"_???

```suggestion
    scenario "admin can view all the submissions", js: true do
```